streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableKTableOuterJoinTest.java

```diff
@@ -75,8 +77,7 @@ public void testJoin() {
         final Collection<Set<String>> copartitionGroups =
             TopologyWrapper.getInternalTopologyBuilder(builder.build()).copartitionGroups();
 
-        assertEquals(1, copartitionGroups.size());
-        assertEquals(new HashSet<>(Arrays.asList(topic1, topic2)), copartitionGroups.iterator().next());
```

Comment:
I'd broken this test at some point, but it didn't print useful information, so I improved the test output. It's not really necessary to include.